{
  "vulinfoid": "JVNDB-2021-020188",
  "vulinfodata": {
    "title": "The Qt Company の Qt における境界外読み取りに関する脆弱性",
    "vulinfodescription": {
      "overview": "The Qt Company の Qt には、境界外読み取りに関する脆弱性が存在します。"
    },
    "affected": {
      "affecteditem": [
        {
          "name": "The Qt Company",
          "productname": "Qt",
          "cpe": {
            "text": "cpe:/a:qt:qt",
            "version": "2.2"
          },
          "versionnumber": [
            "5.15.1",
            "6.0.0",
            "6.0.2",
            "6.2.0"
          ]
        }
      ]
    },
    "impact": {
      "cvss": [
        {
          "version": "3.0",
          "severity": {
            "text": "High",
            "type": "Base"
          },
          "base": "7.1",
          "vector": "CVSS:3.0/AV:L/AC:L/PR:N/UI:R/S:U/C:H/I:N/A:H"
        }
      ],
      "impactitem": {
        "description": "情報を取得される、およびサービス運用妨害 (DoS) 状態にされる可能性があります。"
      }
    },
    "solution": {
      "solutionitem": {
        "description": "ベンダアドバイザリまたはパッチ情報が公開されています。参考情報を参照して適切な対策を実施してください。"
      }
    },
    "related": {
      "relateditem": [
        {
          "type": "advisory",
          "name": "Common Vulnerabilities and Exposures (CVE)",
          "vulinfoid": "CVE-2021-3481",
          "url": "https://www.cve.org/CVERecord?id=CVE-2021-3481"
        },
        {
          "type": "advisory",
          "name": "National Vulnerability Database (NVD)",
          "vulinfoid": "CVE-2021-3481",
          "url": "https://nvd.nist.gov/vuln/detail/CVE-2021-3481"
        },
        {
          "type": "advisory",
          "name": "関連文書",
          "vulinfoid": "lists.debian.org (msg00028)",
          "url": "https://lists.debian.org/debian-lts-announce/2023/08/msg00028.html"
        },
        {
          "type": "advisory",
          "name": "関連文書",
          "vulinfoid": "access.redhat.com (CVE-2021-3481)",
          "url": "https://access.redhat.com/security/cve/CVE-2021-3481"
        },
        {
          "type": "advisory",
          "name": "関連文書",
          "vulinfoid": "bugreports.qt.io (QTBUG-91507)",
          "url": "https://bugreports.qt.io/browse/QTBUG-91507"
        },
        {
          "type": "advisory",
          "name": "関連文書",
          "vulinfoid": "bugzilla.redhat.com (1931444)",
          "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1931444"
        },
        {
          "type": "advisory",
          "name": "関連文書",
          "vulinfoid": "codereview.qt-project.org (+/337646)",
          "url": "https://codereview.qt-project.org/c/qt/qtsvg/+/337646"
        },
        {
          "type": "cwe",
          "name": "JVNDB",
          "vulinfoid": "CWE-125",
          "url": "https://cwe.mitre.org/data/definitions/125.html",
          "title": "境界外読み取り"
        }
      ]
    },
    "history": {
      "historyitem": [
        {
          "historyno": "1",
          "datetime": "2023-09-26T17:28:29+09:00",
          "description": "[2023年09月26日]\\n  掲載"
        }
      ]
    },
    "datefirstpublished": "2023-09-26T17:28:29+09:00",
    "datelastupdated": "2023-09-26T17:28:29+09:00",
    "datepublic": "2021-04-01T00:00:00+09:00"
  }
}
